src/SeatGeek/Sixpack/Session/Base.php

```diff
@@ -157,10 +157,23 @@ protected function getUserAgent()
 
     protected function getIpAddress()
     {
-        if (isset($_SERVER['REMOTE_ADDR'])) {
-            return $_SERVER['REMOTE_ADDR'];
-        }
-        return null;
+    	$ip = null;
+    	if (function_exists('apache_request_headers')) {
+		$headers = apache_request_headers();
+		if (isset($headers['X-Forwarded-For']) && !empty($headers['X-Forwarded-For'])) {
+			$ip = reset(explode(',', $headers['X-Forwarded-For']));
+		} elseif (isset($headers['HTTP_X_FORWARDED_FOR']) && !empty($headers['HTTP_X_FORWARDED_FOR'])) {
+			$ip =  reset(explode(',', $headers['HTTP_X_FORWARDED_FOR']));
+		} else {
+			//
```

Comment:
Yea, force of habit on the empty ifs. Just a personal style thing (intentionally left blank)